content/index.html

```diff
@@ -1,10 +1,13 @@
 ---
-title: BaySec
-description: SF Bay Area Security Professionals Meetup
```

Comment:
We should probably keep this, but change to what you did below, this is useful for search engines.